PowerDocu.SolutionDocumenter/SolutionDocumentationGenerator.cs

```diff
@@ -9,7 +9,7 @@ namespace PowerDocu.SolutionDocumenter
 {
     public static class SolutionDocumentationGenerator
     {
-        public static void GenerateDocumentation(string filePath, string fileFormat, bool documentDefaultChangesOnly, bool documentDefaults, string flowActionSortOrder, string wordTemplate = null)
+        public static void GenerateDocumentation(string filePath, string fileFormat, bool documentDefaultChangesOnly, bool documentDefaults, string flowActionSortOrder, string wordTemplate = null, string outputPath = null)
```

Comment:
Add support for a different path to save the documentation to so that in a DevOps pipeline the documentation can be created in a specific folder on an agent and then commited to the repository.

It becomes hard to extract the documentation out when it's being saved in the folder that the solution is so this is a quality of life feature